source/model/Activator.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABC, abstractmethod
+
+
```

Comment:
Give some comment documentation regarding the purpose of the class. Do this for all non-trivial classes. Not really needed for the endpoints, as you documented them well already.